.github/workflows/unit-tests.yml

```diff
@@ -106,6 +106,7 @@ jobs:
       uses: shivammathur/setup-php@v2
       with:
         php-version: ${{ matrix.php-version }}
+        coverage: none
```

Comment:
This disables XDebug, which a) we don't use and b) interferes with the new OOM tests in some PHP versions